back/app/models/initiative_status.rb

```diff
@@ -33,6 +33,10 @@ class InitiativeStatus < ApplicationRecord
   validates :code, uniqueness: true, unless: :custom?
   validates :description_multiloc, presence: true, multiloc: { presence: true }
 
+  def public?
```

Comment:
We should properly think it through https://citizenlabco.slack.com/archives/C05H2FX706M/p1690911814050209